appmanifest/id-member/id-member-manual-tentative.js

```diff
@@ -0,0 +1,34 @@
+window.onload = () => {
+  const instruction = document.createElement("div");
+  instruction.innerHTML = `
+        <h2>
+          Instructions
+        </h2>
+        <ol>
+          <li>Install <a href="id-member-manual-tentative.html">id-member-manual-tentative.html</a> as a PWA</li>
+          <li>
+            Launch the app from OS. The page should be launched to start_url v1.
+          </li>
+          <li>
+            Close the app window.
+          </li>
+          <li>
+            Trigger a manifest update event.
```

Comment:
Do I have to go to a specific URL to do this? I don't see how this can happen, say, without navigating to the v2 page?

Reply:
* Launch  for v1
* Close app window
* Navigate to the v2 page
* Trigger manifest update event.
  * specific-to-user-agent note - on Chrome, trigger by restarting through chrome://restart & visiting the v2 page afterwards
* Launch the app from OS, validate v2

Reply:
No. The id-member-manual-tentative.js uses v1 manifest that has start_url point to v1 page, and the v1 page uses v2 manifest that has start_url points to v2 page. So you just launch the installed site, which goes to v1, and trigger an update, and it should be launched again to v2 if the test passes

Reply:
Ah, got it!

Then my one request here is that the steps should completely work for Chrome now. So if I follow these, and restart where it says to trigger update, will that all work? Assume I have no knowledge of how this feature / web apps work at all. Maybe we need to add chrome-specific sub-steps to the 'trigger manifest update', like chrome://restart and then visit the page X in a tab?

Reply:
I have the chrome specific instruction in the next paragraph:
On Chrome, trigger the update by restarting the browser through chrome://restart.

Yeah it should completely work for Chrome if the feature is turned on.